libraries/chaindb/LICENSE.md

```diff
@@ -0,0 +1,14 @@
+AntelopeIO/chainbase
```

Comment:
Do we just remove this file?

Reply:
No I don't think so. We can't remove the Steemit copyright notice

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/7cc88c2b653e9b94a9da48408b431de6b2b0e51d

Reply:
I tend to agree with Matt that we should keep it. I think maybe we should put fc's back in.

Reply:
It does feel like fc should go back it now. It might have more defensible to remove it at the time since the base repo license was the same (MIT) and the fc license just say "Respective Authors" which is pretty much implied anyways so it wasn't like we were stripping someone's name from the license.

Reply:
Does that mean we have to put the Business Source License txt above the chainbase/fc MIT license text?

Reply:
That part I'm not confident about; we should ask. In the past we did this,
https://github.com/AntelopeIO/spring/blob/main/libraries/chain/webassembly/runtimes/eos-vm-oc/About%20WAVM
And in a bubble to myself that would be the pattern I'd reach for again. But we should double check

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/de5c92fce62e500f4c29cb07177bb8a733e1fae5